{"uid":"51a63b30be734340","name":"Log out from Trello account","fullName":"tests.regression_ui.test_ui_regression.TestUIRegression#test_log_out","historyId":"9eddea92aa33bc4ffb54b4788932f508","time":{"start":1737839591961,"stop":1737839622506,"duration":30545},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: aborted by navigation: loader has changed while resolving nodes\n  (Session info: chrome=132.0.6834.110)\nStacktrace:\n#0 0x5584628c07ca <unknown>\n#1 0x5584623b82f0 <unknown>\n#2 0x5584623bd031 <unknown>\n#3 0x5584623c0988 <unknown>\n#4 0x5584623c0a13 <unknown>\n#5 0x55846244e4fc <unknown>\n#6 0x55846244e117 <unknown>\n#7 0x55846242c9b2 <unknown>\n#8 0x55846244b3b3 <unknown>\n#9 0x55846242c753 <unknown>\n#10 0x5584623f9baa <unknown>\n#11 0x5584623fadfe <unknown>\n#12 0x55846288b38b <unknown>\n#13 0x55846288f307 <unknown>\n#14 0x558462877e7c <unknown>\n#15 0x55846288fec7 <unknown>\n#16 0x55846285c24f <unknown>\n#17 0x5584628af2f8 <unknown>\n#18 0x5584628af4c0 <unknown>\n#19 0x5584628bf646 <unknown>\n#20 0x7f97ce29ca94 <unknown>\n#21 0x7f97ce329c3c <unknown>","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7efe4265aa50>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"01339ac75a98c3a6235dceef8481744b\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC010\n    @allure.title('Log out from Trello account')\n    def test_log_out(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account'):\n            self.test_login(driver)\n    \n        with allure.step('Log out from Trello account'):\n            WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Account.ACCOUNT_MENU))).click()\n            WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Account.LOG_OU_BUTTON))).click()\n            time.sleep(2)\n            WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Account.LOG_OU_BUTTON))).click()\n        with allure.step('Verify that the user is logged out and located at Trello.com/home page'):\n>           assert WebDriverWait(driver, 15).until(EC.url_to_be('https://trello.com/home'))\n\ntests/regression_ui/test_ui_regression.py:307: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:96: in until\n    value = method(self._driver)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/expected_conditions.py:125: in _predicate\n    return url == driver.current_url\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:474: in current_url\n    return self.execute(Command.GET_CURRENT_URL)[\"value\"]\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:384: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7efe42f4d010>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"aborted by navigation\",\"message\":\"aborted by navigation: loader has chang...\\\\n#19 0x5584628bf646 \\\\u003Cunknown>\\\\n#20 0x7f97ce29ca94 \\\\u003Cunknown>\\\\n#21 0x7f97ce329c3c \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: aborted by navigation: loader has changed while resolving nodes\nE         (Session info: chrome=132.0.6834.110)\nE       Stacktrace:\nE       #0 0x5584628c07ca <unknown>\nE       #1 0x5584623b82f0 <unknown>\nE       #2 0x5584623bd031 <unknown>\nE       #3 0x5584623c0988 <unknown>\nE       #4 0x5584623c0a13 <unknown>\nE       #5 0x55846244e4fc <unknown>\nE       #6 0x55846244e117 <unknown>\nE       #7 0x55846242c9b2 <unknown>\nE       #8 0x55846244b3b3 <unknown>\nE       #9 0x55846242c753 <unknown>\nE       #10 0x5584623f9baa <unknown>\nE       #11 0x5584623fadfe <unknown>\nE       #12 0x55846288b38b <unknown>\nE       #13 0x55846288f307 <unknown>\nE       #14 0x558462877e7c <unknown>\nE       #15 0x55846288fec7 <unknown>\nE       #16 0x55846285c24f <unknown>\nE       #17 0x5584628af2f8 <unknown>\nE       #18 0x5584628af4c0 <unknown>\nE       #19 0x5584628bf646 <unknown>\nE       #20 0x7f97ce29ca94 <unknown>\nE       #21 0x7f97ce329c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1737839591710,"stop":1737839591961,"duration":251},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1737839591961,"stop":1737839591961,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: aborted by navigation: loader has changed while resolving nodes\n  (Session info: chrome=132.0.6834.110)\nStacktrace:\n#0 0x5584628c07ca <unknown>\n#1 0x5584623b82f0 <unknown>\n#2 0x5584623bd031 <unknown>\n#3 0x5584623c0988 <unknown>\n#4 0x5584623c0a13 <unknown>\n#5 0x55846244e4fc <unknown>\n#6 0x55846244e117 <unknown>\n#7 0x55846242c9b2 <unknown>\n#8 0x55846244b3b3 <unknown>\n#9 0x55846242c753 <unknown>\n#10 0x5584623f9baa <unknown>\n#11 0x5584623fadfe <unknown>\n#12 0x55846288b38b <unknown>\n#13 0x55846288f307 <unknown>\n#14 0x558462877e7c <unknown>\n#15 0x55846288fec7 <unknown>\n#16 0x55846285c24f <unknown>\n#17 0x5584628af2f8 <unknown>\n#18 0x5584628af4c0 <unknown>\n#19 0x5584628bf646 <unknown>\n#20 0x7f97ce29ca94 <unknown>\n#21 0x7f97ce329c3c <unknown>","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7efe4265aa50>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"01339ac75a98c3a6235dceef8481744b\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC010\n    @allure.title('Log out from Trello account')\n    def test_log_out(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account'):\n            self.test_login(driver)\n    \n        with allure.step('Log out from Trello account'):\n            WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Account.ACCOUNT_MENU))).click()\n            WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Account.LOG_OU_BUTTON))).click()\n            time.sleep(2)\n            WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Account.LOG_OU_BUTTON))).click()\n        with allure.step('Verify that the user is logged out and located at Trello.com/home page'):\n>           assert WebDriverWait(driver, 15).until(EC.url_to_be('https://trello.com/home'))\n\ntests/regression_ui/test_ui_regression.py:307: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:96: in until\n    value = method(self._driver)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/expected_conditions.py:125: in _predicate\n    return url == driver.current_url\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:474: in current_url\n    return self.execute(Command.GET_CURRENT_URL)[\"value\"]\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:384: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7efe42f4d010>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"aborted by navigation\",\"message\":\"aborted by navigation: loader has chang...\\\\n#19 0x5584628bf646 \\\\u003Cunknown>\\\\n#20 0x7f97ce29ca94 \\\\u003Cunknown>\\\\n#21 0x7f97ce329c3c \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: aborted by navigation: loader has changed while resolving nodes\nE         (Session info: chrome=132.0.6834.110)\nE       Stacktrace:\nE       #0 0x5584628c07ca <unknown>\nE       #1 0x5584623b82f0 <unknown>\nE       #2 0x5584623bd031 <unknown>\nE       #3 0x5584623c0988 <unknown>\nE       #4 0x5584623c0a13 <unknown>\nE       #5 0x55846244e4fc <unknown>\nE       #6 0x55846244e117 <unknown>\nE       #7 0x55846242c9b2 <unknown>\nE       #8 0x55846244b3b3 <unknown>\nE       #9 0x55846242c753 <unknown>\nE       #10 0x5584623f9baa <unknown>\nE       #11 0x5584623fadfe <unknown>\nE       #12 0x55846288b38b <unknown>\nE       #13 0x55846288f307 <unknown>\nE       #14 0x558462877e7c <unknown>\nE       #15 0x55846288fec7 <unknown>\nE       #16 0x55846285c24f <unknown>\nE       #17 0x5584628af2f8 <unknown>\nE       #18 0x5584628af4c0 <unknown>\nE       #19 0x5584628bf646 <unknown>\nE       #20 0x7f97ce29ca94 <unknown>\nE       #21 0x7f97ce329c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","steps":[{"name":"Log in to Trello account","time":{"start":1737839601961,"stop":1737839619421,"duration":17460},"status":"passed","steps":[{"name":"Navigate to the Trello login page","time":{"start":1737839601961,"stop":1737839605313,"duration":3352},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Enter valid credentials (email, password, and 6-digits verification code)","time":{"start":1737839605313,"stop":1737839619421,"duration":14108},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Log out from Trello account","time":{"start":1737839619421,"stop":1737839622065,"duration":2644},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Verify that the user is logged out and located at Trello.com/home page","time":{"start":1737839622066,"stop":1737839622504,"duration":438},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: aborted by navigation: loader has changed while resolving nodes\n  (Session info: chrome=132.0.6834.110)\nStacktrace:\n#0 0x5584628c07ca <unknown>\n#1 0x5584623b82f0 <unknown>\n#2 0x5584623bd031 <unknown>\n#3 0x5584623c0988 <unknown>\n#4 0x5584623c0a13 <unknown>\n#5 0x55846244e4fc <unknown>\n#6 0x55846244e117 <unknown>\n#7 0x55846242c9b2 <unknown>\n#8 0x55846244b3b3 <unknown>\n#9 0x55846242c753 <unknown>\n#10 0x5584623f9baa <unknown>\n#11 0x5584623fadfe <unknown>\n#12 0x55846288b38b <unknown>\n#13 0x55846288f307 <unknown>\n#14 0x558462877e7c <unknown>\n#15 0x55846288fec7 <unknown>\n#16 0x55846285c24f <unknown>\n#17 0x5584628af2f8 <unknown>\n#18 0x5584628af4c0 <unknown>\n#19 0x5584628bf646 <unknown>\n#20 0x7f97ce29ca94 <unknown>\n#21 0x7f97ce329c3c <unknown>\n\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 307, in test_log_out\n    assert WebDriverWait(driver, 15).until(EC.url_to_be('https://trello.com/home'))\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 96, in until\n    value = method(self._driver)\n            ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/expected_conditions.py\", line 125, in _predicate\n    return url == driver.current_url\n                  ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 474, in current_url\n    return self.execute(Command.GET_CURRENT_URL)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":5,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1737839622615,"stop":1737839622694,"duration":79},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"TC010"},{"name":"tag","value":"TC000"},{"name":"parentSuite","value":"tests.regression_ui"},{"name":"suite","value":"test_ui_regression"},{"name":"subSuite","value":"TestUIRegression"},{"name":"host","value":"fv-az1689-679"},{"name":"thread","value":"4093-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.regression_ui.test_ui_regression"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":4,"skipped":0,"passed":106,"unknown":0,"total":110},"items":[{"uid":"e11a7bf4cea4fe66","status":"passed","time":{"start":1737829000409,"stop":1737829031472,"duration":31063}},{"uid":"e4bb31f08720672c","status":"passed","time":{"start":1737817992262,"stop":1737818023254,"duration":30992}},{"uid":"3a33f57fbc2ed3a1","status":"passed","time":{"start":1737807658644,"stop":1737807690896,"duration":32252}},{"uid":"ff9ed51d5af1f2dd","status":"passed","time":{"start":1737796505832,"stop":1737796536888,"duration":31056}},{"uid":"2b4314c3ce0f899","status":"passed","time":{"start":1737785878365,"stop":1737785909522,"duration":31157}},{"uid":"afca60bdd21adae3","status":"passed","time":{"start":1737774928499,"stop":1737774959883,"duration":31384}},{"uid":"d5ba612a9a84962e","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: aborted by navigation: loader has changed while resolving nodes\n  (Session info: chrome=132.0.6834.110)\nStacktrace:\n#0 0x55a851cb77ca <unknown>\n#1 0x55a8517af2f0 <unknown>\n#2 0x55a8517b4031 <unknown>\n#3 0x55a8517b7988 <unknown>\n#4 0x55a8517b7a13 <unknown>\n#5 0x55a8518454fc <unknown>\n#6 0x55a851845117 <unknown>\n#7 0x55a8518239b2 <unknown>\n#8 0x55a8518423b3 <unknown>\n#9 0x55a851823753 <unknown>\n#10 0x55a8517f0baa <unknown>\n#11 0x55a8517f1dfe <unknown>\n#12 0x55a851c8238b <unknown>\n#13 0x55a851c86307 <unknown>\n#14 0x55a851c6ee7c <unknown>\n#15 0x55a851c86ec7 <unknown>\n#16 0x55a851c5324f <unknown>\n#17 0x55a851ca62f8 <unknown>\n#18 0x55a851ca64c0 <unknown>\n#19 0x55a851cb6646 <unknown>\n#20 0x7f449989ca94 <unknown>\n#21 0x7f4499929c3c <unknown>","time":{"start":1737766091859,"stop":1737766122353,"duration":30494}},{"uid":"dfaa2b4956c3851","status":"passed","time":{"start":1737753216683,"stop":1737753247874,"duration":31191}},{"uid":"d2c4aec2a3b1443d","status":"passed","time":{"start":1737742692167,"stop":1737742723389,"duration":31222}},{"uid":"496b0277f32a3c52","status":"passed","time":{"start":1737731730068,"stop":1737731761744,"duration":31676}},{"uid":"8535cd715ae2a64a","status":"passed","time":{"start":1737721359250,"stop":1737721390274,"duration":31024}},{"uid":"e89f980d4bf221a0","status":"passed","time":{"start":1737710130525,"stop":1737710161369,"duration":30844}},{"uid":"ad8d64dab43b1684","status":"passed","time":{"start":1737699516345,"stop":1737699546748,"duration":30403}},{"uid":"d4816dc6524aa4dd","status":"passed","time":{"start":1737688627018,"stop":1737688658637,"duration":31619}},{"uid":"ff3474babb6d0fda","status":"passed","time":{"start":1737679779771,"stop":1737679811087,"duration":31316}},{"uid":"3f29e849ea243944","status":"passed","time":{"start":1737673980545,"stop":1737674011605,"duration":31060}},{"uid":"9407471b527ffbb2","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: aborted by navigation: loader has changed while resolving nodes\n  (Session info: chrome=132.0.6834.110)\nStacktrace:\n#0 0x561b529467ca <unknown>\n#1 0x561b5243e2f0 <unknown>\n#2 0x561b52443031 <unknown>\n#3 0x561b52446988 <unknown>\n#4 0x561b52446a13 <unknown>\n#5 0x561b524d44fc <unknown>\n#6 0x561b524d4117 <unknown>\n#7 0x561b524b29b2 <unknown>\n#8 0x561b524d13b3 <unknown>\n#9 0x561b524b2753 <unknown>\n#10 0x561b5247fbaa <unknown>\n#11 0x561b52480dfe <unknown>\n#12 0x561b5291138b <unknown>\n#13 0x561b52915307 <unknown>\n#14 0x561b528fde7c <unknown>\n#15 0x561b52915ec7 <unknown>\n#16 0x561b528e224f <unknown>\n#17 0x561b529352f8 <unknown>\n#18 0x561b529354c0 <unknown>\n#19 0x561b52945646 <unknown>\n#20 0x7fcc9349ca94 <unknown>\n#21 0x7fcc93529c3c <unknown>","time":{"start":1737672543703,"stop":1737672574144,"duration":30441}},{"uid":"df9018a7bdfeec9f","status":"passed","time":{"start":1737666823554,"stop":1737666854569,"duration":31015}},{"uid":"67ffa055259f740","status":"passed","time":{"start":1737656424077,"stop":1737656455128,"duration":31051}},{"uid":"8153bcfde6e64621","status":"passed","time":{"start":1737645307194,"stop":1737645338252,"duration":31058}}]},"tags":["TC010","TC000"]},"source":"51a63b30be734340.json","parameterValues":[]}